### DISCUSSION TITLE
openxlsx vs openxlsx2

### DISCUSSION THREAD

**Discussion by klar-C** [JanMarvin/openxlsx2#225] [General]
Hi,

Is there a variance table of the features that are not yet included in openxlsx2 vs the old library?

Regards

**Comment by JanMarvin** [comment]
Hi @klar-C ,

welcome to `openxlsx2`. To answer your question, there is no such list or better ... there is such a list (https://github.com/JanMarvin/openxlsx2/issues/34), but it is already very outdated. On the positive side, I don't know anything that `openxlsx2` can't do that its predecessor already could. 

BUT, we are still in the first phase of our public alpha. And we are still collecting feedback and still developing things. The code base has changed a lot and `openxlsx2` is certainly not a "drop in replacement", but as described in the README, a modern reinterpretation. Based on `openxlsx` and many other packages and programming adventures, we were able to approach more directly the task `openxlsx2` is intended to solve and did not have to explore every piece of code. Apart from the `pugixml` and `R6` software projects we are building on, and the heavy and still ongoing task of code streamlining and cleanup, a few things have changed on the user side, most notably the move to a chain or pipe-based approach, and the removal of `stylesObject()`. I wasn't a fan of that and removed it, and lately I've been offering alternative approaches to what `stylesObject()` was supposed to do.

While we are, or at least I am, incredibly proud of what we have already accomplished, this code is not yet ready for the production pipeline. If you want to jump in, we are happy to guide you a bit, but most likely it will take some time before we reach a stable and tested version `1.0`.

PS: Since this is not really a problem, I move this to the discussion.

**Comment by jmbarbone** [comment]
I think it would be good to at least have a minimalized vignette (à la https://cpp11.r-lib.org/articles/converting.html). I don't think we'd need to go overboard.  Maybe we can add some more into #34 to use the updated function names.
